### DISCUSSION TITLE
Hub to Explorer Link (API syntax to open a link to specific STAC Item in Explorer)

### DISCUSSION THREAD

**Discussion by scottyhq** [microsoft/PlanetaryComputer#197] [Ideas]
Right now, Explorer provides code snippets to open in the Hub, but it's not clear if it's possible to go in the other direction (a shareable link to a STAC Item that opens in Explorer). This would be a really amazing feature!

For example, while interactively exploring datasets on the Hub you might come across an interesting acquisition to discuss with someone else. I've been using the `map` endpoint for this, which is really convenient but doesn't allow much exploration. [documented here](https://planetarycomputer.microsoft.com/docs/reference/data/)

https://planetarycomputer.microsoft.com/api/data/v1/item/map?collection=landsat-c2-l2&item=LE07_L2SP_045028_20200317_02_T1

What I'm envisioning is effectively the same URL but opened in Explorer to have access for all the nice UI bells and whistles like different basemaps, exploring scenes near in time, etc. Example URL below, does not actually work:
https://planetarycomputer.microsoft.com/explore?collection=landsat-c2-l2&item=LE07_L2SP_045028_20200317_02_T1

Another nicety would be titiler `viewer` [endpoint](https://developmentseed.org/titiler/endpoints/cog/#viewer), which allows for some basic analysis (different colormaps, histogram, etc). 

Just to clarify there is no issue here, just wanted to propose the idea :) Perhaps it's even already planned? https://planetarycomputer.microsoft.com/docs/overview/explorer/#coming-soon

**Comment by TomAugspurger** [comment]
Thanks for opening this up! I agree there's a lot more that could be done to better tie together the explorer and the results you might get from the STAC / Data API.

One nuanced issue is that the explorer *typically* deals with searches and mosaics of items, rather than individual items. So anything returned in the body of a STAC item might be difficult to incorporate into the Explorer. Even something like the single item you shared results in a URL like `https://planetarycomputer.microsoft.com/explore?c=-120.9320%2C46.0561&z=8.00&v=2&d=landsat-c2-l2&m=cql%3Aecb4021151cef2071b9ada6cd45ce59e&r=Natural+color&s=false%3A%3A100%3A%3Atrue&sr=desc&ae=0`. I think that `cql` parameter indicates a search is involved, but I'm not 100% sure about that.

And just in case it's helpful, https://planetarycomputer.microsoft.com/docs/quickstarts/using-the-data-api/ has some examples of using the `/data` API to construction searches. It's not documented, *Those* URLs should be easier to pass over to the explorer. But they're much harder to construct than getting something off a STAC item.
